pest/src/iterators/pairs.rs

```diff
@@ -287,12 +290,12 @@ mod tests {
         assert_eq!(
             format!("{:?}", pairs),
             "[\
-                Pair { rule: a, span: Span { str: \"abc\", start: 0, end: 3 }, inner: [\
-                    Pair { rule: b, span: Span { str: \"b\", start: 1, end: 2 }, inner: [] }\
-                ] }, \
-                Pair { rule: c, span: Span { str: \"e\", start: 4, end: 5 }, inner: [] }\
-            ]"
-        .to_owned()
+             Pair { rule: a, span: Span { str: \"abc\", start: 0, end: 3 }, inner: [\
```

Comment:
Maybe this should be `rustfmt::skip`?

Reply:
Definitely agree that the string formatting is useful here.